gp_services.R

```diff
@@ -13,23 +13,23 @@ library(RJSONIO)
 services <- fromJSON("https://ual.geoplatform.gov/api/communities/4eebc494059beab9fda54cb078927ddc/items?type=regp:Service&size=200")
 
 #Create an empty vector to hold all of the service IDs
-service_ids <- c()
-service_label <- c() 
+id <- c()
+title <- c() 
 
 #Loop through each service entry in the JSON file (now an R list)
 for(i in 1:length(services[[1]])){
-  service_ids[i] <- services[[1]][[i]][['id']] 
-  service_label[i] <- services[[1]][[i]][['label']] 
+  id[i] <- services[[1]][[i]][['id']] 
+  title[i] <- services[[1]][[i]][['label']] 

```

Comment:
For consistency, is it better to call the variable `label` instead of `title`?

Reply:
We'd like the csv columns and xml tags to say 'title' instead of 'label'. I can change it in places not relating to the column names or tag names, but let me know what you think would be most consistent considering that specification.